### ISSUE TITLE
Most entities defined in the FGD don't exist

### ISSUE THREAD

**Issue by globalcringe** [Zet0rz/nzombies#406]
Out of every entity in the FGD provided, only two entities exist: nz_electricity and nz_zombie_spawn.
The others don't exist:
nz_autoload_config
nz_trigger_buyable
nz_game_listener
nz_carryitem_creator
nz_carryitem_spawner
nz_carryitem_checker

Got a generic error attempting to use any on any map, even with gamemode / map extensions enabled
`Attempted to create unknown entity type nz_autoload_config!
Can't init nz_autoload_config`
`Attempted to create unknown entity type nz_trigger_buyable!
Can't init nz_trigger_buyable`
`Attempted to create unknown entity type nz_game_listener!
Can't init nz_game_listener`
`Attempted to create unknown entity type nz_carryitem_creator!
Can't init nz_carryitem_creator`
`Attempted to create unknown entity type nz_carryitem_spawner!
Can't init nz_carryitem_spawner`
`Attempted to create unknown entity type nz_carryitem_checker!
Can't init nz_carryitem_checker
`

**Comment by jaxjaxk** [comment]
does the gamemode work fine? i have gotten errors like this before and it was basically just due g-mod being g-mod.

**Comment by globalcringe** [comment]
Yeah it works 100% fine, but these entities are defined in the nZombies FGD, something used by mappers to place entities.
The fact that these don't exist on the github means I can't make use of them.

**Comment by Zet0rz** [comment]
Kinda stupid, I forgot I made the .fgd before the entities to get people to see if all the options they have are enough. I'll get to work on the entities missing, meanwhile try testing those that exist (such as nz_electricity) and tell me if there's any input/outputs you want me to add.

**Comment by globalcringe** [comment]
So far: nz_electricity accepts inputs, but doesn't send outputs (like OnElectricityOn or OnElectricityOff)
nz_zombie_spawner exists on the level, but doesn't do anything. Doesn't appear to accept inputs, either.
logic_winlose works kinda (game doesn't immediately end, players can still go down and die after they "win")
logic_waves works fine, with both OnWaveStart and OnWaveEnd firing properly
Files used:
https://feen.us/ure3ysvvio.zip
I'll be testing more entities, but right now I need to get to work

**Comment by globalcringe** [comment]
Every entity works properly, aside from what I've listed.
Also, I really like what you've done with the FGD, many useful things in there.
